lib/js/dom.nim

```diff
@@ -39,6 +39,10 @@ type
     onselect*: proc (event: Event) {.nimcall.}
     onsubmit*: proc (event: Event) {.nimcall.}
     onunload*: proc (event: Event) {.nimcall.}
+    when (NimMajor, NimMinor) >= (1, 3):
```

Comment:
```suggestion
    since (1, 3):
```

Reply:
You can't use templates insode object type definitions, only direct when statements are allowed

Reply:
In the end I had to remove `when` as well. It doesn't work, semfold chokes.